### ISSUE TITLE
Spring Batch & SCDF - Limitations ?

### ISSUE THREAD

**Issue by javaHelper** [mminella/cloud-native-batch#1]
Michael - I'd like to take opportunity to thank for doing great tutorials and videos. I'd request you to do the same for Spring Batch & SCDF more.

I've small concerns on SCDF & Batch 

I learn that SCDF has no support for MongoDB, also the kind of batch jobs that I've is loading data from relational db and loading data to MongoDB, I am not able to monitor batch jobs through SCDF UI and neither see any workaround to save Task enabled Jobs metadata into different schema or something similar.

Could you please guide ion this please ?

**Comment by mminella** [comment]
Spring Batch and Spring Cloud Task both do not support MongoDB as a repository.  There are a number of reasons for that. You'd need to have a relational database as well for SCDF, etc to have visibility into the repository.

**Comment by javaHelper** [comment]
Yeah, Thanks for the details, could you please guide me here : https://stackoverflow.com/questions/61730436/caused-by-java-net-socketexception-connection-reset-in-spring-batch-for-async ? As I'm constantly facing this issue
